### PR TITLE
update for Xcode 8.1

### PR DESCRIPTION
**Pull request by nicksnyder**

### CODE REVIEW

File: LayoutKitTests/ReloadableViewLayoutAdapterCollectionViewTests.swift [hunk 1]
```diff
@@ -64,32 +64,32 @@ private class TestCollectionView: LayoutAdapterCollectionView, TestableReloadabl
         reloadDataCount += 1
     }
 
-    private override func insertItems(at indexPaths: [IndexPath]) {
```
Comment (plivesey) [inline LayoutKitTests/ReloadableViewLayoutAdapterCollectionViewTests.swift]: What's the reason for this change?

Comment (nicksnyder) [inline LayoutKitTests/ReloadableViewLayoutAdapterCollectionViewTests.swift]: Not sure why this worked in Xcode 8.0. It doesn't make sense that you could override a method and make it private :)

```
Overriding instance method must be as accessible as its enclosing type
```
